app/config.js

```diff
@@ -73,11 +77,12 @@ config.js = {
 config.log = {
   requests: process.env.ENABLE_ACCESS_LOGS !== 'false',
   stream: process.stdout,
-  level: process.env.LOG_LEVEL || 'debug',
```

Comment:
? Removing ability to override log level in env?

Reply:
Oops, didn't mean to commit that. `LOG_LEVEL` is also used by the API and so the settings clash and I get no logging from the frontend, so I was temporarily ignoring the env var. I will remove this.